_tools/codespell-ignore-lines.txt

```diff
@@ -1,20 +1 @@
-| ba            | Bashkir                       |
-| dum           | Middle Dutch                  |
-| fo            | Faroese                       |
-| fro           | Old French                    |
-| inh           | Ingush                        |
-| mye           | Myene                         |
-| nd            | North Ndebele                 |
-| ro            | Romanian                      |
-| seh           | Sena                          |
-| shs           | Shuswap                       |
-| te            | Telugu                        |
-| tem           | Timne                         |
-| Beng        | Bengali                     |
-| Maka        | Makasar                     |
-| BA           | Bosnia and Herzegovina                     |
-| FO           | Faroe Islands                              |
-| MM, BU       | Myanmar                                    |
-| RO           | Romania                                    |
-| VN           | Viet Nam                                   |
 .. image:: img/expres.png
```

Comment:
We could probably rename that file to something not truncated and get rid of the ignore lines file altogether, but maybe we'll need it in the future for some other false positives.